packages/engine/src/datastore/batch/migration/mod.rs

```diff
@@ -166,18 +170,28 @@ struct NextState {
 impl<'a> BufferActions<'a> {
     pub fn new_batch(
         &self,
-        schema: &Arc<AgentSchema>,
+        agent_schema: &Arc<AgentSchema>,
+        message_schema: &Arc<MessageSchema>,
         experiment_id: &ExperimentId,
         affinity: usize,
```

Comment:
Considering we're touching this code already, can we rename this to worker_index as affinity has annoyed me for a while and isn't clear at all.

It won't just be this function though so if you can find and replace on the whole code-base for "affinity"

Reply:
I'd say let's do this in another small follow-up PR to keep the history clean.

Reply:
I have `affinity` in my PR anyways

Reply:
#387